tests/integration/test_integration.py

```diff
@@ -159,228 +155,6 @@ def test_firehose_kinesis_to_s3(self):
         all_objects = testutil.list_all_s3_objects()
         testutil.assert_objects(json.loads(to_str(test_data)), all_objects)
 
-    # TODO fix duplication with test_lambda_streams_batch_and_transactions(..)!
-    def test_kinesis_lambda_sns_ddb_sqs_streams(self):
```

Comment:
I deleted this test because it attempts to test various scenarios across multiple different services. Therefore the test itself is difficult to understand and maintain.  Additionally, the content of the test is already covered by other, more discrete tests classes within `test_lambda_integration.py`:

- kinesis event source mappings are already tested in `TestKinesisSource`
- dynamodb streams event source mappings are tested in `TestDynamoDBEventSourceMapping`
- Lambda SNS subscriptions are covered by `TestSNSSubscription`
- SQS event source mappings are covered by `TestSQSEventSourceMapping`